contracts/payloads/IGP35/PayloadIGP35.sol

```diff
@@ -0,0 +1,764 @@
+            false
+        );
+
+        VAULT_T1_FACTORY.setVaultAuth(
+            getVaultAddress(8),
+            0xafE3974f4916140a093F1de7Fc064A3Da220DD41, // Vault_SUSDE_USDT
+            false
+        );
+
+        VAULT_T1_FACTORY.setVaultAuth(
+            getVaultAddress(17),
+            0xa7C805988f04f0e841504761E5aa8387600e430b, // Vault_SUSDE_USDC
+            false
+        );
+
+        VAULT_T1_FACTORY.setVaultAuth(
+            getVaultAddress(18),
+            0x7607968F40d7Ac4Ef39E809F29fADDe34C00A0A6, // Vault_SUSDE_USDT
+            false
+        );
+    }
+
+    /// @notice Action 7: Make sUSDe Vaults borrow rate magnifier 1x
+    function action7() internal {
+        IFluidVaultT1(getVaultAddress(7)).updateBorrowRateMagnifier(100 * 1e2);
+        IFluidVaultT1(getVaultAddress(8)).updateBorrowRateMagnifier(100 * 1e2);
+        IFluidVaultT1(getVaultAddress(17)).updateBorrowRateMagnifier(100 * 1e2);
+        IFluidVaultT1(getVaultAddress(18)).updateBorrowRateMagnifier(100 * 1e2);
+    }
+
+    /// @notice Action 8: Transfer stETH from Treasury to Team Multisig
+    function action8() internal {
+        string[] memory targets = new string[](1);
+        bytes[] memory encodedSpells = new bytes[](1);
+
+        string memory withdrawSignature = "withdraw(address,uint256,address,uint256,uint256)";
+
+        // Spell 1: Transfer stETH
+        {
+            uint256 stETH_AMOUNT = 300 * 1e18; // 300 stETH
+            targets[0] = "BASIC-A";
+            encodedSpells[0] = abi.encodeWithSignature(
+                withdrawSignature,
+                stETH_ADDRESS,
+                stETH_AMOUNT,
+                TEAM_MULTISIG,
+                0,
+                0
+            );
+        }
+
+        IDSAV2(TREASURY).cast(targets, encodedSpells, address(this));
+    }
+
+    /// @notice Action 9: Add new LendingRewards contracts
+    function action9() internal {
+        address[] memory protocols = new address[](2);
+        address[] memory tokens = new address[](2);
+        uint256[] memory amounts = new uint256[](2);
+
+        { /// fUSDC
+            IFTokenAdmin(F_USDC).updateRewards(
+                0x99c1515fa3327B048FCB46483ac0fceB0ED8d471
+            );
+
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDC
+            );
+
+            protocols[0] = F_USDC;
+            tokens[0] = USDC_ADDRESS;
+            amounts[0] = allowance + (205_000 * 1e6);
+        }
+
+        { /// fUSDT
+            IFTokenAdmin(F_USDT).updateRewards(
+                0x99c1515fa3327B048FCB46483ac0fceB0ED8d471
+            );
+
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDT
+            );
+
+            protocols[0] = F_USDT;
+            tokens[0] = USDT_ADDRESS;
+            amounts[0] = allowance + (205_000 * 1e6);
```

Comment:
array is still setting [0] position here when it should be [1] 

and we are not triggering setting the allowance in the end at the reserve contract?